src/AmalgamatorSession.ts

```diff
@@ -71,6 +72,21 @@ export interface RequestArguments extends DebugProtocol.LaunchRequestArguments {
 // eslint-disable-next-line @typescript-eslint/no-empty-interface
 export interface LaunchRequestArguments extends RequestArguments {}
 
+export interface ChildDapRequestArguments {
+    child: number;
```

Comment:
Please add some documentation to what this field and type are.

especially it should say that the value of child should be one of the indexes from a previous call that returned ChildDapContents

Reply:
I deleted it. Because after double-checking, I see no need to use `ChildDapRequestArguments`